### PR TITLE
feat(query): support compute quota feature

### PR DESCRIPTION
**Pull request by zhang2014**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

feat(query): support compute quota feature

- Fixes #[Link the issue here]

## Tests

- [ ] Unit Test
- [x] Logic Test
- [ ] Benchmark Test
- [ ] No Test  - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/15336)

### CODE REVIEW

File: src/common/license/src/license.rs [hunk 1]
```diff
@@ -82,25 +112,97 @@ pub struct LicenseInfo {
     pub r#type: Option<String>,
     pub org: Option<String>,
     pub tenants: Option<Vec<String>>,
-    pub features: Option<Vec<String>>,
+    pub features: Option<Vec<Feature>>,
```
Comment (Xuanwo) [inline src/common/license/src/license.rs:115]: Will this break existing licenses?

Comment (zhang2014) [reply src/common/license/src/license.rs:115]: tested in https://github.com/datafuselabs/databend/pull/15336/files#diff-ecc00d87658b61b7d05f5b675362d6a71d462f76325bca55f4e5f14d95af74cfR144

**Comment by Xuanwo** [comment]
licenses test failed:

```shell
+++ /runner/_work/databend/databend/tests/suites/5_ee/00_check/00_0014_license_info.stdout	2024-04-25 12:23:24.739703048 +0000
@@ -1,3 +1,3 @@
 []
-aggregate_index,background_service,computed_column,data_mask,inverted_index,license_info,storage_encryption,stream,vacuum,virtual_column
+Unlimited
 [(0,)]
```

**Review by Xuanwo — APPROVED** [review]

**Comment by Xuanwo** [comment]
Let's merge. I will rebase my PR.

**Pull request merged**